internal/broadcaster/broadcaster.go

```diff
@@ -104,16 +102,16 @@ func (r *Runner) createAirdropTx(ctx context.Context, participant data.Participa
 		return nil, fmt.Errorf("failed to simulate tx: %w", err)
 	}
 
-	tx, err = r.genTx(ctx, gasUsed*3, participant)
+	tx, err = r.genTx(ctx, gasUsed*3, airdrop)
```

Comment:
Do we need gas multiplier, may be it will be better to make this value configurable?

Reply:
I suggest we leave it until the fees come in place. It came from https://github.com/rarimo/broadcaster-svc/blob/main/internal/services/broadcaster/fee.go#L7 We can't just replace it everywhere, who knows what happens then))